### PR TITLE
Add files via upload

### PR DESCRIPTION
**Pull request by Jingyaqin**

### CODE REVIEW

File: Fraction.java [hunk 1]
```diff
@@ -0,0 +1,28 @@
+import java.io.*;
+public class Fraction{
+    private int numerator;
+    private int denominator;
+
+    public String toString(){
```
Comment (go-keiryo) [inline Fraction.java]: Please fix the intents here.

File: Fraction.java [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+}
+}
+public Fraction(int wholeNumber){
+wholeNumber = numerator;
+denominator = 1;
+}
+public Fraction(String fraction){
+    String fraction = new String();
+    fraction = fraction.trim();
+    int fraction = Integer.parseInt(String);
```
Comment (ghost) [inline Fraction.java:27]: Integer.parseInt(String); !?

File: Fraction.java [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+if(denominator == 0){
+    throw new IllegalArgumentException("denominator is zero");
+}
+if(denominator < 0) {
+        numerator *= -1;
+        denominator *= -1;
+}
+}
+public Fraction(int wholeNumber){
+wholeNumber = numerator;
```
Comment (ghost) [inline Fraction.java:21]: Isn't it numerator = wholeNumber; ?

**Review by ghost — CHANGES_REQUESTED** [review]

**Comment by ghost** [comment]
Please follow the style guide provided from Google. 

[Java Style Guide](https://google.github.io/styleguide/javaguide.html)

File: Fraction.java [hunk 4]
```diff
@@ -0,0 +1,28 @@
...
+public class Fraction{
+    private int numerator;
+    private int denominator;
+
+    public String toString(){
+        int n = numerator;
+        int d = denominator;
+        System.out.println(n/d);
+    }
+public Fraction(int numberator, int denominator){
```
Comment (ghost) [inline Fraction.java:11]: intent is wrong

File: Fraction.java [hunk 5]
```diff
@@ -0,0 +1,28 @@
...
+if(denominator < 0) {
+        numerator *= -1;
+        denominator *= -1;
+}
+}
+public Fraction(int wholeNumber){
+wholeNumber = numerator;
+denominator = 1;
+}
+public Fraction(String fraction){
```
Comment (ghost) [inline Fraction.java:24]: intent is wrong

**Pull request merged**